pubspec.yaml

```diff
@@ -0,0 +1,12 @@
+name: dart2_fix
+description: A tool to migrate API usage to Dart 2.
+version: 1.0.0
```

Comment:
`1.0.0-dev` until we're ready to ship...

Reply:
done